src/metrics-server/templates/metrics-server-deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "metrics-server.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "metrics-server.labels" . | indent 4 }}
```

Comment:
Need these labels on the apiservice too I think.  (So that helm delete may remove the resource)